src/xrp/xrpTypes.ts

```diff
@@ -32,3 +34,11 @@ export const asXrpNetworkLocation = asObject({
   currency: asString,
   issuer: asString
 })
+
+export type SafeXrpWalletInfo = ReturnType<typeof asSafeXrpWalletInfo>
+export const asSafeXrpWalletInfo = asSafeCommonWalletInfo
+
+export type XrpPrivateKeys = ReturnType<typeof asXrpPrivateKeys>
+export const asXrpPrivateKeys = asObject({
+  rippleKey: asString
+})
```

Comment:
This is the one exception to s/Xrp/Ripple/, since everything else is already messed up and using the wrong name anyhow.